<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kass</class>
 <widget class="QDialog" name="Kass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>443</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>752</width>
    <height>443</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>752</width>
    <height>443</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="first_Name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="middle_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Вагоны</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Фамилия</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Отчество</string>
   </property>
  </widget>
  <widget class="QTableView" name="Carriage">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>351</width>
     <height>192</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Sell">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Продать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Value">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Стоимость</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер рейса</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Place_Start">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Пункт назначения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Пункт отправления</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Place_Finish">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Number">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Время отправления</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Time_Start">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата отправления</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="Date_Start">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="calendarPopup">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="days">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Exit</sender>
   <signal>clicked()</signal>
   <receiver>Kass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Carriage</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>Kass</receiver>
   <slot>SetValue(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Sell</sender>
   <signal>clicked()</signal>
   <receiver>Kass</receiver>
   <slot>Sell()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SetValue(QModelIndex)</slot>
  <slot>Sell()</slot>
  <slot>SetValue()</slot>
 </slots>
</ui>
